_pages/en/get-involved.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: Get Involved
 lang: en
+ref:
+  it: /it/come-partecipo
```

Comment:
Temo che questo sia un doppione, ovvero `get-involved` e `how-to-participate` sono la stessa pagina tradotta da due persone diverse (me e @alranel). A questo punto direi di eliminare `how-to-participate.md` e tenere questa. Che ne dite @sebbalex @alranel ?

Reply:
Effettivamente sembra essere doppione, d'accordo.